Scripts/Python/step_response_identification.py

```diff
@@ -0,0 +1,122 @@
+import pandas as pd
+import numpy as np
+from typing import Tuple
+import control
+from control.matlab import step
```

Comment:
Please, use the native function "control.step_response" instead of the MATLAB compatible version "control.matlab.step".